config/cylc_hpc/flow.cylc

```diff
@@ -57,9 +56,13 @@
         """
     [[extractfeatures]]
         script = """
-            julia -t auto $project_dir/workflow/scripts/ice-floe-tracker.jl extractfeatures -i $preprocess_dir -o $features_dir --minarea $minfloearea --maxarea $maxfloearea
+            julia -t auto $project_dir/workflow/scripts/ice-floe-tracker.jl extractfeatures -i $preprocess_dir -o $preprocess_dir --minarea $minfloearea --maxarea $maxfloearea
         """
     [[tracking]]
         script = """
-            julia -t auto $project_dir/workflow/scripts/ice-floe-tracker.jl track --imgs $preprocess_dir --props $features_dir --deltat $preprocess_dir --output $tracker_dir
+            julia -t auto $project_dir/workflow/scripts/ice-floe-tracker.jl track --imgs $preprocess_dir --props $preprocess_dir --deltat $preprocess_dir --output $tracker_dir
+        """
+    [[exportH5]]
+        script = """
+            julia -t auto $project_dir/workflow/scripts/savetoh5.jl --pathtosampleimg $fetchdata_dir/truecolor/20220504.aqua.truecolor.250m.tiff --resdir $preprocess_dir
```

Comment:
I was thinking choosing the image for latlon could be done programmatically.
```suggestion
            julia -t auto $project_dir/workflow/scripts/savetoh5.jl --pathtosampleimg $fetchdata_dir/truecolor/$(ls $fetchdata_dir/truecolor | head -1) --resdir $preprocess_dir
```

That'd get the first hit in the truecolor imgs directory. Do you think the user should be in charge of this? The cli for savetoh5.jl describes this, but for the cycl file this could be taken care of for them.